koku/masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -80,7 +80,7 @@ def get_processing_date(
     return use_cols, time_interval, process_date
 
 
-def create_daily_archives(
+def create_daily_archives(  # noqa C901
```

Comment:
We should work to reduce the complexity of this function.

Reply:
Agreed, we have a tech debt issue that should cover handling this later.